x/posts/internal/types/post.go

```diff
@@ -170,8 +170,8 @@ func (p Post) Validate() error {
 		return fmt.Errorf("invalid post owner: %s", p.Creator)
 	}
 
-	if len(strings.TrimSpace(p.Message)) == 0 {
-		return fmt.Errorf("post message must be non empty and non blank")
+	if len(strings.TrimSpace(p.Message)) == 0 && len(p.Medias) == 0 {
+		return fmt.Errorf("post message cannot be empty nor blank when there are no medias")
```

Comment:
Can change this to make it more generic

```suggestion
		return fmt.Errorf("post message or medias required, they cannot be both empty")
```